### PR TITLE
feat: Shimmer effect in groupwise skills

### PR DESCRIPTION
**Pull request by atm1504**
Fixes #2200 

Changes: 
Instead of the progress bar, a shimmer effect rises in the GroupWise skill listing page.

Screenshots for the change: 
shimmer

### CODE REVIEW

**Comment by ShridharGoel** [comment]
Please resolve conflicts.

**Comment by atm1504** [comment]
@iamareebjamal please review it.

File: app/src/main/res/layout/fragment_group_wise_skill_listing.xml [hunk 1]
```diff
@@ -18,6 +18,35 @@
...
+            android:id="@+id/groupWiseSkillsShimmerContainer"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="vertical">
+                <!-- & rows of shimmer added-->
```
Comment (ShridharGoel) [inline app/src/main/res/layout/fragment_group_wise_skill_listing.xml]: I think you meant 7 rows

**Comment by atm1504** [comment]
@iamareebjamal please review it

**Comment by mariobehling** [comment]
This kind of area is low priority at the moment, but ok merging it here. Please focus on the actual features we need to make the app work from here on.

**Pull request merged**